CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing to Aloxide
+
+1. [Getting Involved](#getting-involved)
+2. [How To Develop Features](#how-to-develop)
+3. [How To Report Bugs](#how-to-report-bugs)
+
+## Getting Involved
+
+We're always looking for help [implementing features](#how-to-develop), [identifying bugs](#how-to-report-bugs), writing test cases, and improving documentation. If you find any issues, please don't hesitate to discuss with us, any ideas are helps for us.
+
+There're some related guides for you to check before contributing to our project:
+- [High-level Architecture](https://github.com/lecle/aloxide/wiki/High-level-Architecture)
+
+## How To Develop features
+
+### Changing existing/planning features
+
+Make sure you already take a deep look at our [High-level Architecture](https://github.com/lecle/aloxide/wiki/High-level-Architecture). We, as an on-going project, welcome anyone help us improve our code, ideas or design.
+
+### Adding new ones
+
+It would be great to hear new ideas, features contribute to Aloxide, if you have any, don't hesitate to let us know about that.
+
+## How To Report Bugs
+
+### Send us your concerns
+
+Any contributions are precious to us, if you have any troubles or just have some ideas to improve Aloxide, please don't hesitate to send us your concerns at our groups https://groups.google.com/g/aloxide, we're happy and willing to help checking at anytime when we're available.
+
+### Try the latest version of Aloxide
+
+Bugs in old versions of Aloxide may have already been fixed. In order to avoid reporting known issues, make sure you are always testing against the latest build.
+
```

Comment:
Please also do a quick search in the [issues](link here) page to see if your issue has been raised by others.